### PR TITLE
Machine file changes for PNNL's Sooty

### PR DESCRIPTION
**Pull request by singhbalwinder**
This machine is an old machine but it is still useful.

[BFB] - Bit-For-Bit

### CODE REVIEW

**Comment by singhbalwinder** [comment]
@jayeshkrishna : This machine has an old compiler so PIO 2 (SCORPIO) doesn't work on this machine. I have set PIO version for this machine as 1 in the machine files. Is that an okay place to set machine defaults?

File: cime_config/machines/config_pio.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <entry id="PIO_VERSION">
     <values>
       <value>2</value>
+      <value mach="sooty">1</value>
```
Comment (singhbalwinder) [inline cime_config/machines/config_pio.xml:8]: @jayeshkrishna : This is where I have set PIO to be 1 for this machine.

**Review by jayeshkrishna — APPROVED** [review]
Looks good to me. Did you test the changes on another machine (compy)? Just to make sure that the change (PIO version change for ex) does not affect other machines.

**Comment by singhbalwinder** [comment]
Thanks @jayeshkrishna . I did a quick test on Compy and the selected PIO version was 2 (which is what we expect). I think this PR is ready to go and I will merge it when the repo is open.

**Comment by rljacob** [comment]
This can be merged.  No impact on WC

**Comment by singhbalwinder** [comment]
Thanks @rljacob . I have merged it to next.

**Pull request merged**